### PR TITLE
Add React Native Example

### PR DESCRIPTION
**Pull request by ex7r3me**
This PR include an example app for emoji-mart-native, related to #6

### CODE REVIEW

**Comment by pederjohnsen** [comment]
@ex7r3me thanks :+1:

I'll try and get this reviewed before next week :ok_hand:

**Comment by ex7r3me** [comment]
Nice 👌

**Comment by pederjohnsen** [comment]
@ex7r3me just a heads up, I'm about to release a new version `v0.4.0-beta` with some more changes, most notably the changes to the local image requires. Just need to do some more testing on it in a production app before pushing the release :)

**Comment by pederjohnsen** [comment]
@ex7r3me just released `v0.4.0-beta`, local image requires changes:
https://github.com/thinkproductivity/emoji-mart-native#local-image-requires

it also introduces a breaking change by renaming `showBackButton` => `showCloseButton`

please use this version now instead :+1:

**Comment by ex7r3me** [comment]
@pederjohnsen I've updated the sample app with the latest version, can you check the changes?
here is a screenshot of current status:
screen

**Comment by pederjohnsen** [comment]
@ex7r3me Thanks! I'll try and check the changes today :ok_hand:

File: example/App.js [hunk 1]
```diff
@@ -4,46 +4,97 @@
  * @flow
  */
 
-import React, { Component } from 'react';
+import React, { Component } from 'react'
+import { Platform, StyleSheet, Text, View, Button, Picker } from 'react-native'
```
Comment (pederjohnsen) [inline example/App.js]: @ex7r3me looks like none of `Platform/Button/Picker` is being used in this file, so let's remove them :)

File: example/App.js [hunk 2]
```diff
@@ -4,46 +4,97 @@
...
+        />
+        <ModalPicker
+          isVisible={this.state.modalVisible}
+          showCloseButton={true}
+          onPressClose={() => {
+            this.showPickerTrigger(false)
+          }}
+          set={this.state.set}
+          data={data}
+          onSelect={this.emojiSelectTrigger}
```
Comment (pederjohnsen) [inline example/App.js]: @ex7r3me we should also close the modal picker after selecting an emoji with it

**Comment by pederjohnsen** [comment]
@ex7r3me seems like the `<Picker/>` doesn't work very well on iOS..


Maybe we should consider just using buttons like in the emoji-mart example?

The search/skin selection bar on the picker can be ignored, seems like an issue with the library itself :O

File: example/App.js (hunk 1 above)
Comment (ex7r3me) [reply example/App.js]: Thanks, they have been removed

File: example/App.js (hunk 2 above)
Comment (ex7r3me) [reply example/App.js]: Good point, now it closes on select trigger :+1:

**Comment by ex7r3me** [comment]
@pederjohnsen oops that picker doesn't look good on iOS! 
I think our options are:
- going with buttons (simplest but might not be very pretty)
- playing with style and find a way to fix overlapping picker and Emoji (might be time-consuming, and not the best result as the action sheet looks better for this case)
- use action sheet for iOS  (looks like the best experience to me but might get codebase complex for having a different component for each platform)
- using a third-party component like [this](https://github.com/yfuks/react-native-action-sheet) to handle the cross-platform problem (looks simplest but add a dependency and might not works well)
I suggest the 4th option but as I don't have access to a Mac right now I'm afraid I can't test options 2,3. I just wanted to know all the options but we still can go with fit-to-text buttons and don't add complexity.

**Comment by pederjohnsen** [comment]
@ex7r3me I'm ok with going for the 4th option for now, we can always look to change it later on :+1:

File: example/App.js [hunk 3]
```diff
@@ -32,7 +33,19 @@ export default class App extends Component {
   render() {
     return (
       <View style={styles.container}>
-        <Text style={styles.titleText}>Emoji Mart Native</Text>
+        <View style={styles.titleContainer}>
+          <Text style={styles.titleText}>Emoji Mart Native</Text>
+          <Emoji emoji="department_store" size={16} />
```
Comment (johnsenpeder) [inline example/App.js]: @ex7r3me I've had a little more think about this one, let's remove this `<Emoji/>` and replace `santa` with `department_store` above :+1:

Comment (ex7r3me) [reply example/App.js]: @pederjohnsen Alright, sorry for the delay, I had an intense week with multiple interviews which happily went good, I can get back on the sample and will make the changes ASAP.

**Comment by pederjohnsen** [comment]
@ex7r3me you still able to push through on this or should I take over the last bit? :)

**Comment by ex7r3me** [comment]
@pederjohnsen I thinks it's better if you can take over from here, I'm in the middle of migrating to a new country for the job and it took more time and complexity than I expected. 
The last thing that I was working on was to create a demo for both simple button and third party lib but just implemented the first one. I've pushed the simple button solution, I hope it helps and I'll be back when I get stable.

**Comment by pederjohnsen** [comment]
@ex7r3me no problem, thanks for all the work so far and good luck with the job and migrating to a new country :raised_hands:

**Pull request merged**